src/sentry/issues/occurrence_consumer.py

```diff
@@ -261,11 +261,13 @@ def _process_message(
         sampled=True,
     ) as txn:
         try:
-            if message.get("payload_type") == PayloadType.OCCURRENCE.value:
-                return process_occurrence_message(message, txn)
-            elif message.get("payload_type") == PayloadType.STATUS_CHANGE.value:
+            if message["payload_type"] == PayloadType.STATUS_CHANGE.value:
```

Comment:
Hmm, actually this will break any messages without a payload type right?

Reply:
We should attempt to fetch the payload type, and if not present assume it's occurrence